modules/tinymce/src/core/main/ts/api/dom/DOMUtils.ts

```diff
@@ -1716,15 +1713,6 @@ const DOMUtils = (doc: Document, settings: Partial<DOMUtilsSettings> = {}): DOMU
      */
     findCommonAncestor,
 
-    /**
-     * Parses the specified RGB color value and returns a hex version of that color.
-     *
-     * @method toHex
-     * @param {String} rgbVal RGB string value like rgb(1,2,3)
-     * @return {String} Hex version of that RGB value like #FF00FF.
-     */
-    toHex,
```

Comment:
This needs a TinyMCE changelog entry to say we removed this API. For example something like this:
```
- Removed the `toHex` function for the `DOMUtils` and `Styles` APIs #TINY-8163
```

- [ ] Add a TinyMCE changelog entry for the removed public APIs.

Reply:
We'll also need an entry in the `Changed` section to say that we no longer normalise all colours to hex. e.g
```
- RGB colors are no longer converted to hex values when parsing or serializing content #TINY-8163
```